.github/CODEOWNERS

```
@@ -253,8 +253,10 @@ README.md @sqs
 /cmd/frontend/db/discussion* @slimsag
 
 # LSIF
-/cmd/frontend/internal/httpapi/lsif.go @sourcegraph/code-intel
 /lsif/ @sourcegraph/code-intel
+/cmd/frontend/graphqlbackend/*lsif* @sourcegraph/code-intel
```

Do you think it is more precise?

```suggestion
/cmd/frontend/graphqlbackend/*lsif_* @sourcegraph/code-intel
```